plugin/pkg/scheduler/algorithm/priorities/util/topologies.go

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
@davidopp - I splitted non_zero.go file into two parts - this is mostly copy of that with some small changes.